### PR TITLE
T9044 Initial translation method

### PR DESCRIPTION
**Pull request by Toxicom**
Issue #192
https://app.forecast.it/project/P-59/workflow/T9044

# Summary of Changes

1. Add two translation JSON's for each language
2. Add a helper function that returns correct translated string based on pathname
3. Use `getTranslatedText()` for translations

## Unrelated changes
1. Remove unused `altHeader` propType from `Header.js`

# Notes

Tried to accomplish this with https://www.gatsbyjs.com/plugins/gatsby-plugin-react-i18next/ but ran into problems, for now this temporary solution will do just fine.

# To test

- [ ] Go to `/informacija-ukrainieciams/migracijos-paslaugos/#tab-0`
- [ ] See if copy button's text is correct
- [ ] Go to `/ua/refugee-guide/migration-office/#tab-0`
- [ ] See if copy button's text is correct

### CODE REVIEW

File: src/helpers/handlers.js [hunk 1]
```diff
@@ -39,3 +41,12 @@ export const isUkrainianPage = () => {
   const pathname = useLocation().pathname;
   return pathname.includes(`/ua/`);
 };
+
+export const getTranslatedText = (path) => {
```
Comment (bmarty92) [inline src/helpers/handlers.js:45]: move this helper function to separate file (utils dir)

File: src/helpers/handlers.js [hunk 2]
```diff
@@ -39,3 +41,12 @@ export const isUkrainianPage = () => {
...
+  const translations = isUkrainianPage() ? UA : LT;
+  // reduce() referenced from stackoverflow
+  // https://stackoverflow.com/a/43849204
+  return path.split(`.`).reduce((prevValue, currValue) => {
```
Comment (bmarty92) [inline src/helpers/handlers.js:49]: return value should be variable

Comment (Toxicom) [reply src/helpers/handlers.js:49]: Could you explain why?

File: src/helpers/handlers.js (hunk 1 above)
Comment (Toxicom) [reply src/helpers/handlers.js]: https://github.com/adaptdk/su-ukraina/pull/198/commits/b71faf36fcf9b22e3367ca5b993363b12cebf167

**Review by IamRokas — APPROVED** [review]

**Pull request merged**